.cicd/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+. $HELPERS_DIR/execute.sh
+
+execute mkdir -p $BUILD_DIR
+
+if [[ $(uname) == Darwin ]]; then
+
+    MAC_TEST="ctest -j$JOBS --output-on-failure -T Test"
```

Comment:
What is the advantage to making these BASH commands a variable that is then evaluated for execution?
Let's hard-code the ctest commands, because they should not be changing and they aren't meant to be configurable.